src/features/FeatureExtractor.js

```diff
@@ -377,17 +378,21 @@ class FeatureExtractor {
      * @param {*} dumpLineObj
      */
     _recordSessionDuration(dumpLineObj) {
-        const [ requestType, , , timestamp ] = dumpLineObj;
+        const [ requestType, , requestObj, timestamp ] = dumpLineObj;
 
         if (requestType !== 'connectionInfo' && requestType !== 'identity') {
-            if (!this.conferenceStartTime && timestamp) {
-                this.conferenceStartTime = timestamp;
+            if (!this.sessionStartTime && timestamp) {
+                this.sessionStartTime = timestamp;
             }
 
-            if (timestamp > this.conferenceEndTime) {
-                this.conferenceEndTime = timestamp;
+            if (timestamp > this.sessionEndTime) {
+                this.sessionEndTime = timestamp;
             }
         }
+
+        if (requestType === 'conferenceStartTimestamp' && requestObj) {
```

Comment:
can this be moved to a separate handle function e.g. handleConferenceBla...?

Reply:
I tried that, but then setting `this.conferenceStartTime` wasn't visible from outside the function call any more. That is why I reverted to this less ideal solution.

Reply:
Ah. Very cool. Thanks that pointer helped me resolve this issue.